Sources/UIComponent/Components/View/Text.swift

```diff
@@ -77,7 +77,7 @@ public struct Text: ViewComponent {
         } else {
             // Faster route
             let size = attributedString.boundingRect(with: constraint.maxSize,
-                                                     options: [.usesLineFragmentOrigin],
```

Comment:
Can we make this an option on the `Text` component(similar to `lineBreakMode`)? Since this change might break existing app's layout.

We also need to set it in the first `if` case.
```swift
            layoutManager.usesFontLeading = false // set this value
```